site/sync.js

```diff
@@ -1,25 +1,32 @@
 const path = require('path')
-const fs = require('fs-extra')
-const config = require('./config')
+const _fs = require('fs')
 const { promisify } = require('util')
-const readdirP = promisify(fs.readdir)
-const statP = promisify(fs.stat)
-const readFile = promisify(fs.readFile)
-const writeFile = promisify(fs.writeFile)
-const deleteFile = promisify(fs.unlink)
+
+const config = require('./config')
+
+// When Node.js 8 support is dropped we can just remove this
+// and change the _fs above to fs = require('fs').promises
+const fs = {
+  copyFile: promisify(_fs.copyFile),
+  readdir: promisify(_fs.readdir),
+  stat: promisify(_fs.stat),
+  readFile: promisify(_fs.readFile),
+  writeFile: promisify(_fs.writeFile),
+  unlink: promisify(_fs.unlink),
+}
 
 async function readDir(dir, allFiles = []) {
-  const files = (await readdirP(dir)).map(f => path.join(dir, f))
+  const files = (await fs.readdir(dir)).map(f => path.join(dir, f))
   allFiles.push(...files)
-  await Promise.all(files.map(async f => (await statP(f)).isDirectory() && readDir(f, allFiles)))
+  await Promise.all(files.map(async f => (await fs.stat(f)).isDirectory() && readDir(f, allFiles)))
   return allFiles
 }
 
 async function syncLocalContent() {
   const src = path.join(config.docs.srcPath)
   const destination = path.join(config.docs.outputPath)
 
-  await fs.copy(src, destination)
+  await fs.copyFile(src, destination)
```

Comment:
⚠ This is not recursive like the one from `fs-extra`, so will not really work. I'm open to suggestions on how to handle this.